keyboards/handwired/evk/v1_3/config.h

```diff
@@ -39,13 +39,13 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
  *
  */
 #define MATRIX_ROW_PINS \
-    { B0, B1, B2, B3, B7, D0 }
+    { B0, B1, B2, B3, B7, D0 }  // row 0, 1, 2, ...
```

Comment:
```suggestion
    { B0, B1, B2, B3, B7, D0 }
```

Reply:
Hi, would it be possible to retain these comments to assist potential readers?  
https://github.com/YangPiCui/ErgonomicVerticalKeyboard/tree/master/EVK_v1.3/ElectronicsFirmwareAndSoftware#121-handwiredevkv1_3

Reply:
I'm not sure what value it adds, it should be fairly self-explanatory.

Reply:
Alright.